src/bidiMapper/CommandProcessor.ts

```diff
@@ -234,7 +234,6 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEvents> {
         );
       } else {
         const error = e as Error;
-        console.error(error);
```

Comment:
> Remove a forgotten console.error. If you think we should use the logger here, happy to make the change.

Considering we already have the `logger` in the constructor, it should be easy to call it here. You can store it as a private member variable in the constructor. Let's do that.